src/main/java/pl/task/parkingmeter/controller/VehicleController.java

```diff
@@ -0,0 +1,190 @@
+package pl.task.parkingmeter.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import pl.task.parkingmeter.entity.Profit;
+import pl.task.parkingmeter.entity.Rates;
+import pl.task.parkingmeter.entity.Vehicle;
+import pl.task.parkingmeter.exception.InvalidDateException;
+import pl.task.parkingmeter.exception.InvalidRegNumberException;
+import pl.task.parkingmeter.exception.VehicleAddedEarlierException;
+import pl.task.parkingmeter.exception.VehicleNotFoundException;
+import pl.task.parkingmeter.repository.RatesService;
+import pl.task.parkingmeter.repository.ValueRateService;
+import pl.task.parkingmeter.repository.VehicleService;
+
+import java.math.BigDecimal;
+import java.sql.Timestamp;
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+@RestController
+@RequestMapping("/")
+@EnableWebMvc
+public class VehicleController {
+
+    private VehicleService vehicleService;
+    private RatesService ratesService;
+    private ValueRateService valueRateService;
+
+    @Autowired
+    public VehicleController(VehicleService vehicleService, RatesService ratesService, ValueRateService valueRateService) {
+        this.vehicleService = vehicleService;
+        this.ratesService = ratesService;
+        this.valueRateService = valueRateService;
+    }
+
+
+    @GetMapping("")
+    public List<Vehicle> showAllVehicles() {
+        return vehicleService.findVehiclesByPaidFalse();
```

Comment:
wyjmujesz obiekt z DB i 1:1 zwracasz go na frontend
Jakie to może powodować problemy?

Reply:
problemem może być przesłanie jakiś wrażliwych danych, jeżeli występują